dist/obs-server.spec

```diff
@@ -427,7 +427,7 @@ getent passwd obsservicerun >/dev/null || \
 if [ -f /etc/sysconfig/obs-server ] ; then
     . /etc/sysconfig/obs-server
 fi
-for i in deltastore dispatcher dodup warden ; do
+for i in deltastore dispatcher dodup obsgetbinariesproxy publisher rep_server servicedispatch signer src_service src_server warden ; do
```

Comment:
bs_service is running as "obsservicerun" user, so src_service needs also to be owned by "obsservicerun" - I checked our own intance and have seen that the logfile is owned by root. Is there something changing with systemd?

Reply:
yes, that's the point. all services running under systemd need their logfiles owned by the user they are running as.